### ISSUE TITLE
More children of abnormal tumor vascularization

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3172]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)
1) abnormal number of tumor pericytes
2) increased tumor nonattached pericytes
3) decreased tumor nonattached pericytes
4) increased tumor attached pericytes
5) decreased tumor attached pericytes

>2. Synonyms (e.g., absent spleen)
1) ...adventitial reticular cells; ...Cells of Rouget; ...Pericytes of Rouget;...ARCs
2-5) ...adventitial reticular cells; ...Cells of Rouget; ...Pericytes of Rouget;...ARCs

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).
1) An anomaly in the average number of pericytes counted in a sampled perivascular area of a tumor.
2) An increase in the number of nonattached pericytes counted in a sampled perivascular area of a tumor.
3) A reduction in the number of nonattached pericytes counted in a sampled perivascular area of a tumor.
4)  An increase in the number of attached pericytes counted in a sampled perivascular area of a tumor.
5) A reduction in the number of attached pericytes counted in a sampled perivascular area of a tumor.
PMID:16416090

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)
1) abnormal tumor vascularization (MP:0010144)
2) abnormal number of tumor pericytes (new term)
3) abnormal number of tumor pericytes (new term)
4) abnormal number of tumor pericytes (new term)
5) abnormal number of tumor pericytes (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)
RGD:cp
